### DISCUSSION TITLE
🐈

### DISCUSSION THREAD

**Discussion by jasonjgardner** [jasonjgardner/jg-rtx#29] [Feedback]
![Screenshot 2021-04-28 080729](https://user-images.githubusercontent.com/1903667/116409005-09520e80-a7f9-11eb-92c7-2c859b112fb7.png)

**Comment by erich666** [comment]
The cat as the barrier block is awesome.

![image](https://user-images.githubusercontent.com/803995/116417408-3a3a3f80-a809-11eb-8b35-938c48bced83.png)
